### PR TITLE
Feature/frontend base

### PR DESCRIPTION
**Pull request by conoren**
link working

### CODE REVIEW

File: back-end/back-end/wwwroot/lib/bootstrap/LICENSE [hunk 1]
```diff
@@ -0,0 +1,22 @@
+The MIT License (MIT)
```
Comment (akosgdcsi) [inline back-end/back-end/wwwroot/lib/bootstrap/LICENSE:1]: Luckily you guys pushed this otherwise you couldn't use bootstrap

**Review by akosgdcsi — APPROVED** [review]
Nice work guys✅, backend looks shity, I would say something bad about the frontend, but I'm not a girl to understand it 😾♿️

**Comment by tsaskater** [comment]
We should start over my main stack is angular and this is not angular...
 I'm not happy about it...  Let's just use ngx-bootstrap  [https://valor-software.com/ngx-bootstrap/#/](https://www.youtube.com/watch?v=o-YBDTqX_ZU)

**Comment by IIB3nceII** [comment]
React vs angular is like porsche vs dacia

File: back-end/back-end/wwwroot/lib/bootstrap/LICENSE (hunk 1 above)
Comment (IIB3nceII) [reply back-end/back-end/wwwroot/lib/bootstrap/LICENSE:1]: Idk what happened here

**Review by IIB3nceII — APPROVED** [review]
It took my breath away

**Comment by devv77** [comment]
As it seems to me. It's just too big for to process it alone. If you need any help, or just to go through it again. I am reachable this evening. @IIB3nceII  and we should ask @conoren  to help us out here. Because I believe it's good but we need to understand this change. 
And for @conoren  please try to commit and PR more times, and try to avoid this big of a PR. It looks like it's one big project. 
Thanks. I'll appreciate it.

**Comment by tsaskater** [comment]
> As it seems to me. It's just too big for to process it alone. If you need any help, or just to go through it again. I am reachable this evening. @IIB3nceII  and we should ask @conoren  to help us out here. Because I believe it's good but we need to understand this change. 
> And for @conoren  please try to commit and PR more times, and try to avoid this big of a PR. It looks like it's one big project. 
> Thanks. I'll appreciate it. 

Úristen very big

**Review by Broy1 — APPROVED** [review]
Approved, very nice.

**Pull request merged**